src/Cache/Cache.php

```diff
@@ -168,7 +168,21 @@ protected static function _buildEngine($name)
         }
 
         $config = static::$_config[$name];
-        $registry->load($name, $config);
+
+        try {
+            $registry->load($name, $config);
+        } catch (RuntimeException $e) {
+            if (!array_key_exists('fallback', $config)) {
+                $registry->set($name, new NullEngine());
+                trigger_error($e->getMessage(), E_USER_WARNING);
+
+                return;
+            }
+            $fallbackEngine = clone static::engine($config['fallback']);
+            $copyableConfig = ['groups' => null, 'prefix' => null];
+            $fallbackEngine->init(array_intersect_key($config, $copyableConfig));
```

Comment:
There are essentially two configs going on here, the static one in the cache frontend `Cache::$_groups` and what the engine needs (group, prefix) to create the proper prefixes to actually find the file. 

`Cache::$_groups` needs to know that even though "test" fails, if you reference a group it had it will still write (using the fallback). The fallback engine then needs to use those group configs to construct the proper prefixes.

This dual config caused me a bit of confusion; perhaps I need more tests for us to be totally confident about them?